include/admin.jsonnet

```diff
@@ -85,7 +85,16 @@ local common = import "common.jsonnet";
                       {
                         "name": "CONFIGURATION_SERVICE_PORT",
                         "value": "5672"
-                      }]),
+                      },
+                      {
+                        "name": "API_SERVER_HOSTNAME",
```

Comment:
So I think ADDRESS_CONTROLLER_HOST is wrong in that sense. ADDRESS_SERVICE_HOST is better, but it's not only about addresses anymore... But i agree _HOST is better than _HOSTNAME